llvm/test/Transforms/InstCombine/fcmp.ll

```diff
@@ -2266,6 +2296,17 @@ define i1 @fcmp_sqrt_zero_uno(half %x) {
   ret i1 %cmp
 }
 
+; Make sure that ninf is cleared.
+define i1 @fcmp_sqrt_zero_uno_fmf(half %x) {
+; CHECK-LABEL: @fcmp_sqrt_zero_uno_fmf(
+; CHECK-NEXT:    [[CMP:%.*]] = fcmp ult half [[X:%.*]], 0xH0000
+; CHECK-NEXT:    ret i1 [[CMP]]
+;
+  %sqrt = call half @llvm.sqrt(half %x)
```

Comment:
Missing type suffixes on all the sqrt calls.

Also, can you keep ninf if the sqrt is also ninf?

Reply:
Done.